HermesProxy/World/Server/PacketHandlers/ItemHandler.cs

```diff
@@ -201,10 +201,14 @@ void HandleCancelTempEnchantment(CancelTempEnchantment cancel)
         void HandleWrapItem(WrapItem item)
         {
             WorldPacket packet = new WorldPacket(Opcode.CMSG_WRAP_ITEM);
-            packet.WriteUInt8(item.GiftBag);
-            packet.WriteUInt8(item.GiftSlot);
-            packet.WriteUInt8(item.ItemBag);
-            packet.WriteUInt8(item.ItemSlot);
+            byte GiftBag = item.GiftBag != Enums.Classic.InventorySlots.Bag0 ? ModernVersion.AdjustInventorySlot(item.GiftBag) : item.GiftBag;
+            byte GiftSlot = item.GiftSlot == Enums.Classic.InventorySlots.Bag0 ? ModernVersion.AdjustInventorySlot(item.GiftSlot) : item.GiftSlot;
```

Comment:
You should be checking GiftBag here.

Reply:
What?
What does "checking GiftBag" mean?

Reply:
I have tested wrapping items in the bank and regular inventory with these changes present and everything worked exactly as expected. What do I need to check in addition to that?

Reply:
Go look at the examples i linked... The slot will never be equal to Enums.Classic.InventorySlots.Bag0, that's the main bag. It seems you don't understand what the code is doing. The slot only needs to be corrected if the bag is the main bag which includes equipped items, bag slots, key ring slots, and bag slots. If the bag slot is a different bag, then slots don't need to be adjusted at all.

Reply:
I literally copy-pasted the code exactly as it was from your examples and just changed the variable names, tho?